examples/pre-67u3/cluster/cluster.yaml

```diff
@@ -25,6 +25,7 @@ spec:
     global:
       secretName: "cloud-provider-vsphere-credentials"
       secretNamespace: "kube-system"
+      insecure: true
```

Comment:
Hi @andrewsykim,

I don't think this is required in the pre-67u3 config.

Reply:
good catch, updated